Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Merge.swift

```diff
@@ -18,7 +18,8 @@ extension Docc {
         public init() {}
         
         public static var configuration = CommandConfiguration(
-            abstract: "Merge a list of documentation archives into a combined archive."
+            abstract: "Merge a list of documentation archives into a combined archive.",
+            usage: "docc merge <archive-path> ... [[--landing-page-catalog <catalog-path>] | [<synthesized-landing-page-options>]] [--output-path <output-path>]"
```

Comment:
From https://github.com/swiftlang/swift-docc/pull/1011/files#diff-090d5c64cdfc0f4fdff7b75a5ec658e2eae3edf27ca3795cfba9ef2d9a504822R80 I understand that passing a landing page catalog is not supported yet, right? In that case I think we should remove the landing page catalog parameter from the CLI for now.